components/chainhook-cli/src/scan/stacks.rs

```diff
@@ -245,40 +242,55 @@ pub async fn scan_stacks_chainstate_via_rocksdb_using_predicate(
             apply: hits_per_blocks,
             rollback: vec![],
         };
-        match handle_stacks_hook_action(trigger, &proofs, &ctx) {
+        let res = match handle_stacks_hook_action(trigger, &proofs, &ctx) {
             Err(e) => {
                 error!(ctx.expect_logger(), "unable to handle action {}", e);
+                Ok(()) // todo: should this error increment our err_count?
```

Comment:
@lgalabru, another sanity check - previously we were just logging this error. Do we want this error to possibly get propagated to the user via an `Interrupt` status?

Reply:
you mean updating redis?
that state machine might become a bit tricky to handle but if you're confident and if we have tests, let's do it.

Reply:
Created #412